### ISSUE TITLE
[Bug]: Autosleep timer engaging outside of set time.

### ISSUE THREAD

**Issue by iconoclasthero** [advplyr/audiobookshelf-app#1268]
### What was the Problem?

The sleep timer was set (1200-0600, later changed to 0000-0600).  Playback was started ca. 11:00, outside of either time range.  The sleep timer was still active.

### Steps to Reproduce the Issue

1. Set sleep timer
2. Start playback
3. Observer sleep timer on.

### What was Expected?

Inactive sleep timer

### Phone Model

OnePlus 10 Pro

### Phone OS

Android 14, stock rom, rooted

### Audiobookshelf App Version

Android App - 0.9.74

### Installation Source

Other (List in "Additional Notes")

### Additional Notes

I probably sideloaded it last time?  Not 100% sure.  I know that I _have_ sideloaded it before.

**Comment by iconoclasthero** [comment]
![Screenshot_2024-07-25-11-30-55-9 (1)](https://github.com/user-attachments/assets/67b2e7b8-2ed4-4b6c-a414-cfeea9783525)
![Screenshot_2024-07-25-11-30-39-6 (1)](https://github.com/user-attachments/assets/5c5143d0-8ba4-4824-aa7c-782798b47c4f)

**Comment by nichwall** [comment]
I was able to reproduce this by setting an auto sleep timer to end 3 minutes in the future and beginning playback (observed auto sleep timer enabled correctly). I closed the media player but kept the app open until I was outside of active hours for the auto sleep timer. Opening the player incorrectly enabled the auto sleep timer with full time remaining.

Completely closing the app and reopening the app correctly did not cause the auto sleep timer to be enabled. Looks like once the auto sleep timer is enabled it is not checked whether it should be disabled without completing closing the app.

**Comment by iconoclasthero** [comment]
Yeah, what he said about completely closing the app seemed to work.  I used app info > force stop to close it.
Changing books, logging into the server didn't have any effect.
